.github/workflows/_buildpacks-release-update-builder.yml

```diff
@@ -66,6 +66,7 @@ jobs:
         with:
           token: ${{ steps.generate-token.outputs.app_token }}
           title: Update ${{ github.repository }} to v${{ inputs.buildpack_version }}
+          commit-message: Update ${{ github.repository }} to v${{ inputs.buildpack_version }}
```

Comment:
Actually, should this be a multi-line commit message, with the title set to what you have here now, and then after a newline the body set to `${{ steps.generate-changelog.outputs.changelog }}`?